src/components/core/PivotTable.tsx

```diff
@@ -414,6 +420,51 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private startWatchingTableRendered = () => {
+        // table having no data will be unmounted, it causes ref null
+        if (!this.containerRef) {
+            this.stopWatchingTableRendered();
+            return;
+        }
+
+        // table has data and takes place of Loading icon
+        if (!this.isTableHidden()) {
+            this.stopWatchingTableRendered();
+            setTimeout(this.props.afterRender, 100);
```

Comment:
It seem that it want to waiting for something finished but don't know what. Please consider some how to get rid of this time out.

Reply:
Test and removed timeout.